experiments/AMIP/modular/Project.toml

```diff
@@ -50,10 +50,9 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 ArgParse = "1.1"
 ArtifactWrappers = "0.2"
 AtmosphericProfilesLibrary = "0.1"
-ClimaAtmos = "0.15.1"
 CLIMAParameters = "0.4, 0.5, 0.6, 0.7"
-ClimaComms = "0.3, 0.4"
-ClimaCore = "0.10.40"
+ClimaComms = "0.4, 0.5"
+ClimaCore = "0.10"
```

Comment:
These are specified in the top-level Project's compats, so they don't need to also be specified here. It might just be personal preference, but I think it's cleaner to only have each compat in one place (the highest-level Project it appears in ). This makes it easier to avoid inconsistent compat entries, but you could alternatively just use CompatDevTools.jl's `synchronize_compats` for that.

Reply:
I followed the ClimaAtmos' model. I saw that they have some repeated entries from the top-level Project to the `examples/` one. @charleskawczynski any preference?

Reply:
It's a good point. What matters most is that there's a valid compat entry for every dependency, whether that exists in the examples or package environment is a maintenance matter. I'm fine with removing compats that exist in the main package environment, but then we'll need to keep tabs on which ones exist. We could try to extend Aqua to add tests that a compat entry exists for all deps across multiple environments (that would be ideal IMO).

Reply:
For now, I'm agnostic to keeping / removing them. I'd say it's maybe best left to the maintainer to decide?

Reply:
Okay I removed them